### PR TITLE
remove remaining six dependencies

### PR DESCRIPTION
**Pull request by Pesc0**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [x]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

File: src/Mod/Path/Path/Base/Util.py [hunk 1]
```diff
@@ -162,14 +161,14 @@ def clearExpressionEngine(obj):
 
 def toUnicode(string):
     """toUnicode(string) ... returns a unicode version of string regardless of the python version."""
-    return six.text_type(string)
+    return str(string)
```
Comment (chennes) [inline src/Mod/Path/Path/Base/Util.py:164]: Better to eliminate the toUnicode function entirely--if you don't want to do it here, either leave a TODO or file an issue so we don't forget.

**Review by chennes — APPROVED** [review]

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9067">pipeline status</a> for feature branch [PR_9067](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9067). Pipeline [818372632](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/818372632) was triggered at [42cb756](https://github.com/FreeCAD/FreeCAD/pull/9067/commits/42cb7565223ac602ab21ecfb41cac8c70156ee98). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Path/Path/Base/Util.py (hunk 1 above)
Comment (Pesc0) [reply src/Mod/Path/Path/Base/Util.py:165]: done

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9067">pipeline status</a> for feature branch [PR_9067](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9067). Pipeline [818379632](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/818379632) was triggered at [017b2b7](https://github.com/FreeCAD/FreeCAD/pull/9067/commits/017b2b7f2dfbcd228c00a7c14812264d5e4b500c). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by chennes** [comment]
Fast work, thanks!

**Pull request merged**